include/npcomp/Dialect/ATen/GeneratedATenBuiltinOps.td

```diff
@@ -0,0 +1,2117 @@
+  let arguments = (ins
+    ATenAnyTensor:$input,
+    ATenAnyTensor:$other,
+    ATenAnyRefTensor:$out
+  );
+  let results = (outs
+    ATenAnyRefTensor:$result
+  );
+}
+
+def ATen_BuiltinMulOp: ATenImmutableTensorOp<"builtin.mul", [NoSideEffect]> {
+  let summary = "mul(input, other, out=None)";
+  let arguments = (ins
+    ATenAnyTensor:$input,
+    ATenAnyTensor:$other
+  );
+  let results = (outs
+    ATenAnyTensor:$result
+  );
+  let description = [{
+    Multiplies each element of the input :attr:`input` with the scalar
+    :attr:`other` and returns a new resulting tensor.
+
+    .. math::
+        \text{out}_i = \text{other} \times \text{input}_i
+
+    If :attr:`input` is of type `FloatTensor` or `DoubleTensor`, :attr:`other`
+    should be a real number, otherwise it should be an integer
+
+    Args:
+        {input}
+        value (Number): the number to be multiplied to each element of :attr:`input`
+        {out}
+
+    Example::
+
+        >>> a = torch.randn(3)
+        >>> a
+        tensor([ 0.2015, -0.4255,  2.6087])
+        >>> torch.mul(a, 100)
+        tensor([  20.1494,  -42.5491,  260.8663])
+
+    .. function:: mul(input, other, out=None)
+
+    Each element of the tensor :attr:`input` is multiplied by the corresponding
+    element of the Tensor :attr:`other`. The resulting tensor is returned.
+
+    The shapes of :attr:`input` and :attr:`other` must be
+    :ref:`broadcastable <broadcasting-semantics>`.
+
+    .. math::
+        \text{out}_i = \text{input}_i \times \text{other}_i
+
+    Args:
+        input (Tensor): the first multiplicand tensor
+        other (Tensor): the second multiplicand tensor
+        out (Tensor, optional): the output tensor.
+
+    Example::
+
+        >>> a = torch.randn(4, 1)
+        >>> a
+        tensor([[ 1.1207],
+                [-0.3137],
+                [ 0.0700],
+                [ 0.8378]])
+        >>> b = torch.randn(1, 4)
+        >>> b
+        tensor([[ 0.5146,  0.1216, -0.5244,  2.2382]])
+        >>> torch.mul(a, b)
+        tensor([[ 0.5767,  0.1363, -0.5877,  2.5083],
+                [-0.1614, -0.0382,  0.1645, -0.7021],
+                [ 0.0360,  0.0085, -0.0367,  0.1567],
+                [ 0.4312,  0.1019, -0.4394,  1.8753]])  
+  }];
+}
+
+def ATen_BuiltinMulOutrefOp: ATenRefTensorOp<"builtin.mul_outref", []> {
+  let summary = "See variant of op that does not take an out= ref tensor.";
+  let arguments = (ins
+    ATenAnyTensor:$input,
+    ATenAnyTensor:$other,
+    ATenAnyRefTensor:$out
+  );
+  let results = (outs
+    ATenAnyRefTensor:$result
+  );
+}
+
+def ATen_BuiltinRemainderOp: ATenImmutableTensorOp<"builtin.remainder", [NoSideEffect]> {
+  let summary = "remainder(input, other, out=None) -> Tensor";
+  let arguments = (ins
+    ATenAnyTensor:$input,
+    ATenAnyTensor:$other
+  );
+  let results = (outs
+    ATenAnyTensor:$result
+  );
+  let description = [{
+    Computes the element-wise remainder of division.
```

Comment:
idea (low priority): If there's a python module that can easily do it, converting these to Markdown would be useful so our ODS doc gen stuff works.

Reply:
added todo